src/app/Task3/task3.js

```diff
@@ -0,0 +1,25 @@
+// Recursion
+// We’ve seen that % (the remainder operator) can be used to test whether a
+// number is even or odd by using % 2 to check whether it’s divisible by two.
+// Here’s another way to define whether a positive whole number is even or odd:
+// • Zero is even.
+// • One is odd.
+// • For any other number N, its evenness is the same as N - 2.
+// Define a recursive function isEven corresponding to this description. The function should accept
+//a number parameter and return a Boolean.
+
+// Test it on 50 and 75. See how it behaves on -1. Why? Can you think of a
+// way to fix this?
+function isEven(number) {
+  number < 0 ? (k = -1) : (k = 1);
+
+  return number == 0 || number == 1 ? !number : isEven(number - 2 * k);
+}
+
+// or
+
+function isEven1(number) {
+  return Math.abs(number) == 0 || Math.abs(number) == 1
+    ? !Math.abs(number)
+    : isEven(Math.abs(number) - 2);
+}
```

Comment:
I would rather not to over-complicate things. There is such a term, "function complexity" used  at QA level, which shows a number of operations/loops/variations your code would/could follow while running. The smaller the number, the better code you are to deliver.
I would suggest To use `Math.abs()` to reduce redundant check for negativeness and then multiply at each iteration. Then at ternary operator level I would check for number being less then 2 - at this point you will need to only return equality to 0, otherwise call of the function. Will reduce amount of equality operators to run.
This would possibly be the shortest and easiest way.